bin/main.asm

```diff
@@ -40,13 +40,16 @@ trap:
 
 	mov cx, TRAP_SIZE
 	rep movsb
+	sub di, TRAP_SIZE
 	
 	push ds
```

Comment:
@ItamarAbeliovich Does the swapping of es and ds must be inside the loop? It slows us down